### DISCUSSION TITLE
Tidak bisa menautkan perangkat

### DISCUSSION THREAD

**Discussion by azmirizkifar20** [kirimwa-id/forum#117] [Tanya Jawab] [answered]
![WhatsApp Image 2022-05-22 at 23 26 33](https://user-images.githubusercontent.com/43155964/169705622-ef15a33b-8f94-472e-b6c4-96598e5bec0d.jpeg)
Saya mau coba untuk menautkan perangkat lagi, tapi setiap saya coba scan qr codenya, selalu muncul pop up "Harap pindai kode QR lagi.". kira-kira kenapa yaa?

terimakasih sebelumnya...

**Comment by melodiazz** [comment, marked as answer]
https://github.com/kirimwa-id/forum/discussions/114

**Reply by azmirizkifar20** [reply]
oalahh, okey terimakasih mas
